README.md

```diff
@@ -1,10 +1,74 @@
-# Reti
+<h1 align="center">Reti</h1>
+<div align="center">
+Reti is a in development calculator that can evaluate LaTeX expressions.
+<br>
+<img src="./dark_icon.svg" width="300" />
+</div>
 
-Reti is a calculator that can evaluate LaTeX expressions.
+<h2 align="center" id="development-status">Development Status</h2>
 
+**Note:** Reti is currently under active development. For the latest and potentially unstable features, please check out the [`dev`](https://github.com/barbariand/Reti/tree/dev) branch.
+
+- Current Phase: The project is actively under development with newer, experimental features being added in the dev branch.
+- Stability: Users interested in the most up-to-date but possibly unstable builds should refer to the dev branch.
+
+<h2 align="center">Table of Contents</h2>
+
+- [Development Status](#development-status)
+- [Installation](#installation)
+- [REPL Usage](#repl-usage)
+- [Features](#features)
+- [Contribute](#contribute)
+- [Contributors](#contributors)
+- [Dependencies](#dependencies)
+- [License](#license)
+
+<h2 align="center" id="installation">Installation</h2>
+To get started with Reti, clone the repository and build the project using Rust's package manager, Cargo. Follow these steps:
+
+```bash
+# Clone the repository
+git clone https://github.com/barbariand/Reti.git
+
+# Change directory for the repl
+cd Reti/parser
+
+# Build the project
+cargo build --release
+```
+
+<h2 align="center" id="repl-usage">REPL Usage</h2>
+
+
+Just input a string from any valid latex and get back the correct result
```

Comment:
Hopefully the result is correct, but I think emphasizing that the result is calculated might be better here.
```suggestion
Just input a string from any LaTeX mathematics and get back the calculated result
```